% This file was created (at least in part) by the script ParseMdtoLatex by Louis du Plessis
% (Available from https://github.com/taming-the-beast)

\documentclass[11pt]{article}
\input{preamble}

% Add your bibtex library here
\addbibresource{refs}


%%%%%%%%%%%%%%%%%%%%
% Do NOT edit this %
%%%%%%%%%%%%%%%%%%%%
\begin{document}
\renewcommand{\headrulewidth}{0.5pt}
\headsep = 20pt
\lhead{ }
\rhead{\textsc {BEAST v2 Tutorial}}
\thispagestyle{plain}


%%%%%%%%%%%%%%%%%%
% Tutorial title %
%%%%%%%%%%%%%%%%%%
\begin{center}

	% Enter the name of your tutorial here
	\textbf{\LARGE Tutorial using BEAST v2.5.2}\\\vspace{2mm}

	% Enter a short description of your tutorial here
	\textbf{\textcolor{mycol}{\Large Fast relaxed clock model}}\\

	\vspace{4mm}

	% Enter the names of all the authors here
	{\Large {\em Rong Zhang and Alexei Drummond}}
\end{center}

This tutorial introduces ConstantDistance operator to improve the efficiency BEAST analysis under relaxed clock models. The proposal kernel in the presented operator changes evolutionary rates and divergence times at the same time, under the constraint that the implied genetic distances remain constant. Specifically, the proposal operates on the divergence time of an internal node and the three adjacent branch rates. For the root of a phylogenetic tree, there are three strategies included, named Simple Distance, Small Pulley and Big Pulley. The descriptions below will provide guidance step by step to estimate evolutionary rates and divergence times using BEAST2 software.

%%%%%%%%%%%%%%%%%
% Tutorial body %
%%%%%%%%%%%%%%%%%

\section{Introduction}\label{introduction}
Bayesian phylogenetic inference via MCMC is computationally intensive for large data sets. Two approaches to improve efficiency are (i) by making faster likelihood calculations, and (ii) by incorporating more effective proposal kernels. However, the overall efficiency of the sampling process also depends strongly on the construction of the proposal mechanism. An effective proposal mechanism is proficient at exploring the posterior distribution, and can do so with fewer steps in the MCMC chain. Therefore fewer likelihood calculations are required, since each step in the chain requires a likelihood calculation.

The operator presented in this tutorial, ConstantDistance operator,  is based on an uncorrelated relaxed clock model, in which the rates vary from branch to branch, drawn independently and identically from a given prior distribution, such as the log-normal distribution (\cite{drummond2006relaxed}). As a result, the rates can vary markedly between parent and child branches.

More specifically, the presented operator searches within a subspace of constant genetic distances. Namely, the proposed operator changes both divergence times of nodes and neighbouring branch rates so that implied genetic distances are not changed. For time-reversible substitution models the phylogenetic likelihood will also be unchanged under this operation.

\section{Required programs}\label{programs}
\begin{itemize}
\item BEAST2 - Bayesian Evolutionary Analysis Sampling Trees

BEAST2 is a software platform for phylogenetic analysis based on Bayesian MCMC algorithms. This tutorial is written for BEAST v2.5.0 (\cite{bouckaert2014beast}).
\item BEAUti - Bayesian Evolutionary Analysis Utility

BEAUti is a graphical user interface tool to generate XML files that BEAST2 takes as input. It provides users with configuration options that are necessary for Bayesian phylogenetic analysis, such as prior distributions of parameter and various evolutionary models.
\end{itemize}

You may also need the following programs to analyse the output files of BEAST2.
\begin{itemize}
\item Tracer

This program analyses .log file which records the sampled states in Bayesian MCMC chain by BEAST2 (\url{http://tree.bio.ed.ac.uk/software/tracer}). It displays distributions of sampled parameters and calculates Effective Sample Size (ESS) automatically.
\item FigTree

This program reads .trees file and visualises trees sampled during MCMC chain (\url{http://tree.bio.ed.ac.uk/software/figtree}). The users can designate phylogenetic properties to show on the plotted trees, such as node ages, rates, branch length and so on.
\item TreeAnnotator

This program summaries trees in .tree file based on posterior distribution and gives a maximum clade credibility tree (\url{https://beast2.blogs.auckland.ac.nz/treeannotator/}). 
\end{itemize}

\section{Practical analysis}\label{analysis}
\subsection{Set up BEAUti}
\begin{enumerate}
\item Start BEAUti
\item Import alignment
\item Specify clock model
\item Specify priors
\item Specify operators
\item Set up MCMC chain
\end{enumerate}

\subsection{Run XML file in BEAST2}
\begin{enumerate}
\item Start BEAST2
\end{enumerate}

\subsection{Analyse results}
\begin{enumerate}
\item Analyse .log file
\item Analyse .trees file
\end{enumerate}

\section{Acknowledgment}\label{acknowledgment}

The content of this tutorial is based on the \href{https://github.com/CompEvol/MultiTypeTree/wiki/Beginner's-Tutorial-(short-version)}{StructuredCoalescent tutorial} by Tim Vaughan.

\section{Useful Links}\label{useful-links}
\begin{itemize}
\item
  \href{http://www.beast2.org/book.html}{Bayesian Evolutionary Analysis
  with BEAST 2} \citep{BEAST2book2014}
\item
  \href{https://github.com/denisekuehnert/bdmm}{Multi-type birth-death
  process package} \citep{Kuhnert2016}
\item
  BEAST 2 website and documentation: \url{http://www.beast2.org/}
\end{itemize}

%%%%%%%%%%%%%%%%%%%%%%%
% Tutorial disclaimer %
%%%%%%%%%%%%%%%%%%%%%%%
% Please do not change the license
% Add the author names and relevant links
% Add any other aknowledgments here
\href{http://creativecommons.org/licenses/by/4.0/}{\includegraphics[scale=0.8]{figures/ccby.pdf}} This tutorial was written by Rong Zhang and   Alexei Drummond for \href{https://taming-the-beast.github.io}{Taming the BEAST} and is licensed under a \href{http://creativecommons.org/licenses/by/4.0/}{Creative Commons Attribution 4.0 International License}. 

%%%%%%%%%%%%%%%%%%%%
% Do NOT edit this %
%%%%%%%%%%%%%%%%%%%%
Version dated: \today



\newpage 
%%%%%%%%%%%%%%%%
%  REFERENCES  %
%%%%%%%%%%%%%%%%

\printbibliography[heading=relevref]


\end{document}
